src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -89,10 +89,6 @@ def _get_commits_committer(self, commits, author_id):
         committer_commit_list = [
             serialize(commit) for commit in commits if commit.author.id == author_id
         ]
-
```

Comment:
i cant be asked to make a mapping of this -- we can clean up and make this endpoint more efficient later